code/modules/clothing/shoes/boots.dm

```diff
@@ -68,6 +68,9 @@
 /obj/item/clothing/shoes/jackboots/fast
 	slowdown = -1
 
+/obj/item/clothing/shoes/jackboots/sec
```

Comment:
Joy, now we need to move our own sec overwrite to these boots instead of the ones we made. Curse youu